### DISCUSSION TITLE
Display Title only

### DISCUSSION THREAD

**Discussion by KyleMcCarley** [hopollo/OBS-Youtube-Player#5] [Ideas]
It'd be great for music playlists if there was a way to have the video itself hidden, but still display the title. And bonus points if it could fade in for a few seconds at the start and end of each track.

**Comment by hopollo** [comment]
Hi,

Your request is now available with the param "&title=true"

In your case (**only title**) : &title=true&hide=true
_you also might need to use_ (**on OBS**) property -> **custom css** : **#info { color: white; font-weight: bold; }**

commit : https://github.com/hopollo/OBS-Youtube-Player/commit/7f7762bf7447b8bbd3143c8daa0c36c75160d57e

Have fun

**Reply by KyleMcCarley** [reply]
Thanks! Is there a way to turn off the "by such-and-such" part? In my use case, I'm doing a playlist of game music, and it seems weird to attribute credit to the person who uploaded the track to their YT channel as if they wrote the music

**Reply by hopollo** [reply]
Hi,

Sorry for behing late, i made a small change to allow people to fully edit how the info (title &/or author) should be displayed.

In your case you will have to use those css rules on obs 

#songTitle {
  color: red;
  font-weight: bold;
}

#songAuthor {
  display: none;
}

have fun
